backend/routes/performance_routes.py

```diff
@@ -1,8 +1,25 @@
 from flask import Blueprint
 from flask import jsonify
+from flask import request
+from flask import abort
+from ..db.models import Adjudication
 
 blueprint = Blueprint('performance', __name__, url_prefix='/performances')
 
 @blueprint.route('/')
 def main():
-	return 'Welcome to Dancefest Backend!'
\ No newline at end of file
+	return 'Welcome to Dancefest Backend!'
+
+@blueprint.route('/<performance_id>/adjudications', methods=['GET', 'POST'])
+def handle_adjudications(performance_id):
```

Comment:
i think this is confusing, lets make each method type get its own function.

Reply:
do you mean having the decorator specifically be for the 'GET' method and another decorator referring to the 'POST' method. Then with their own method for each?